libraries/launcher/org/apache/logging/log4j/core/impl/Log4jContextFactory.java

```diff
@@ -0,0 +1,35 @@
+package org.apache.logging.log4j.core.impl;
+
+import org.apache.logging.log4j.core.LoggerContext;
+import org.apache.logging.log4j.spi.LoggerContextFactory;
+
+import java.net.URI;
+/**
+ * This class is a stub and is needed to compile the log4j injector
+ * Changing it may break things
+ **/
+public class Log4jContextFactory implements LoggerContextFactory {
+
+        @Override
+        public LoggerContext getContext(String fqcn, ClassLoader loader, boolean currentContext) { 
+            throw new RuntimeException("This is a stub");
```

Comment:
```suggestion
            throw new UnsupportedOperationException();
```
A message isn't necessary since this code won't be reached.

Reply:
All of them are changed to UnsupportedOperationException